### PR TITLE
Remove Identity from Polkadot Relay

### PR DESCRIPTION
**Pull request by joepetrowski**
Since the launch of the People Chain, we can remove Identity and Migration code.

### CODE REVIEW

**Review by acatangiu — APPROVED** [review]

File: system-parachains/people/people-polkadot/src/lib.rs [hunk 1]
```diff
@@ -110,9 +110,17 @@ pub type SignedExtra = (
...
 
+parameter_types! {
+	pub const IdentityMigratorPalletName: &'static str = "IdentityMigrator";
+}
+
 /// Migrations to apply on runtime upgrade.
 pub type Migrations = (
 	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
+	frame_support::migrations::RemovePallet<
+		IdentityMigratorPalletName,
```
Comment (kianenigma) [inline system-parachains/people/people-polkadot/src/lib.rs:121]: Is this a bomb? how much data will it try and remove at once?

**Review by kianenigma — APPROVED** [review]

File: system-parachains/people/people-polkadot/src/lib.rs (hunk 1 above)
Comment (joepetrowski) [reply system-parachains/people/people-polkadot/src/lib.rs:121]: The only state this has is `StorageVersion`.

File: relay/polkadot/src/lib.rs [hunk 2]
```diff
@@ -1717,6 +1648,214 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
...
+		pallet_proxy::Pallet<T>,
+		Twox64Concat,
+		AccountId,
+		PrevProxiesValue<T>,
+		ValueQuery,
+	>;
+
+	pub struct Migration;
+	impl OnRuntimeUpgrade for Migration {
+		/// Compute the expected post-upgrade state for Proxies stroage, and the reserved value
```
Comment (joepetrowski) [inline relay/polkadot/src/lib.rs:1712]:
```suggestion
		/// Compute the expected post-upgrade state for Proxies storage, and the reserved value
```

File: relay/polkadot/src/lib.rs [hunk 3]
```diff
@@ -1109,7 +1050,7 @@ pub enum ProxyType {
 	Governance = 2,
 	Staking = 3,
 	// Skip 4 as it is now removed (was SudoBalances)
-	IdentityJudgement = 5,
+	// Skip 5 as it was IdentityJudgement
```
Comment (ggwpez) [inline relay/polkadot/src/lib.rs:1053]: This would make proxies undecodable? Probably no issue since we dont need those anymore but CI should complain...

**Review by ggwpez — APPROVED** [review]

**Pull request merged**